content/docs/user-guide/experiment-management/experiments-overview.md

```diff
@@ -0,0 +1,67 @@
+# DVC Experiments Overview
+
+DVC Experiments are captures automatically by DVC when you [run them]. Each
+experiment creates and tracks a project variation based on the changes in your
+<abbr>workspace</abbr>. Experiments preserve the latest commit in the current
+branch (Git `HEAD`) as their parent or _baseline_, but do not form part of the
+regular Git tree or workflow (unless you make them [persistent]).
+
+<details>
+
+### ⚙️ How does DVC track experiments?
+
+Experiments are custom [Git references] (found in `.git/refs/exps`) with a
+single commit based on `HEAD` (not checked out by DVC). Note that these commits
```

Comment:
Experiments with checkpoints have multiple commits.

> (not checked out by DVC)

What's not checked out by DVC? `HEAD`? The experiment?

Reply:
It might be good to mention the motivation for using custom refs and some basic analogy or description of what they are. There are some good examples in [#2979#discussion_r757848241](https://github.com/iterative/dvc.org/pull/2979#discussion_r757848241).

Reply:
Clarified.

Reply:
Not sure we need motivation for implementation details.

Reply:
p.s. I expanded on the general motivation a little based on the link. See d2c01119.